snp_motion_planning/src/planning_server.cpp

```diff
@@ -329,12 +372,15 @@ class PlanningServer
   }
 
   rclcpp::Node::SharedPtr node_;
+  // Internal second ROS node that is used for the Tesseract monitor
+  rclcpp::Node::SharedPtr internal_node_;
```

Comment:
Why can't `node_` also be used for the visualization?

Reply:
I remember the was a reason at one point that I did it this way, but let me double check that it doesn't work with your suggestion.